app/Offer/OfferControllerProvider.php

```diff
@@ -48,8 +50,11 @@ public function connect(Application $app): ControllerCollection
         $controllers->put('/{offerId}/status/', UpdateStatusRequestHandler::class);
         $controllers->put('/{offerId}/booking-availability/', UpdateBookingAvailabilityRequestHandler::class);
 
-        $controllers->put('/{cdbid}/type/{typeId}/', "{$controllerName}:updateType");
-        $controllers->put('/{cdbid}/theme/{themeId}/', "{$controllerName}:updateTheme");
+        $controllers->post('/{offerId}/videos', AddVideoRequestHandler::class);
```

Comment:
This will need to be registered with a trailing slash after you merge in the latest commits from master. (All routes should be registered with a trailing slash since a recent commit.)
It will keep working with and without trailing slash from the outside though, so the acceptance tests that use `/videos` (no trailing slash) for example should keep working.

Reply:
Sorry for that, I did a wrong merge and forgot about the trailing slash.